llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -442,6 +454,9 @@ InstructionCost RISCVTTIImpl::getShuffleCost(TTI::ShuffleKind Kind,
     return LT.first *
            getRISCVInstructionCost(RISCV::VSLIDEDOWN_VI, LT.second, CostKind);
   case TTI::SK_InsertSubvector:
+    if (Index == 0 && any_of(Args, UndefValue::classof))
```

Comment:
Do we need to check that Args isn't empty?

Reply:
Added, thanks!